src/scripts/ibm-openapi-validator.d.ts

```diff
@@ -0,0 +1,21 @@
+declare module "ibm-openapi-validator" {
```

Comment:
Sad that it's not TS.

Reply:
Can we contribute this to @DefinitelyTyped?

Reply:
24 stars on github, I've tried but without a lot of hope 😅 I will contribute to definitelyTyped to add this later.